### ISSUE TITLE
Add event.key values for special characters

### ISSUE THREAD

**Issue by davidnagli** [davidnagli/Scratch-JS#53]
When the user uses whenKeyPressed and attempts to use event.keyCode, it should be replaced with the text equivalent of special keys.

```javascript
whenKeyPressed{
    event.key == "up" //true when up arrow pressed
   event.key == "space" //true when spacebar pressed
}
```

It should work with the following special keys:
* Enter
* Left
* Right
* Up
* Down
* Space
* Backspace

**Comment by skistaddy** [comment]
What about the rest of the keys?

**Comment by davidnagli** [comment]
The rest of the keys are automatically set by JavaScript

**Comment by skistaddy** [comment]
K.

**Comment by skistaddy** [comment]
Solved in #57
